reconcile/utils/terrascript_aws_client.py

```diff
@@ -2451,6 +2455,49 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
             '${' + cf_oai_tf_resource.id + '}'
         tf_resources.append(Output(output_name_0_13, value=output_value))
 
+        # aws_s3_bucket_acl
+        values = common_values.get('distribution_config', {})
+        if 'logging_config' in values.keys():
+            values = {}
+            values['name'] = identifier
+            values['tags'] = common_values['tags']
+
+            aws_s3_bucket_acl_resource = aws_s3_bucket_acl(identifier, **values)
+            tf_resources.append(aws_s3_bucket_acl_resource)
+
+            access_control_policy = {
+                'grant': {
+                    'grantee': {
+                        'id': '${' + aws_s3_bucket_acl_resource.id + '}',
+                        'type': '${' + aws_s3_bucket_acl_resource.type + '}'
+                    },
+                    'permission': 'READ'
+                }
+            }
+            grant = {
+                'grantee': '${' + aws_s3_bucket_acl_resource.grantee + '}',
+                'permission': '${' + aws_s3_bucket_acl_resource.permission + '}'
+            }
+            owner = {
+                'id': '${' + aws_s3_bucket_acl_resource.id + '}'
+            }
```

Comment:
The config above doe not match the example from https://github.com/fmasuhr/terraform-aws-cloudfront-logs/blob/599d948859586dfdaf5189d3be0b1f573cc4be02/s3.tf#L13-L39 . Is that normal ?